releases/v0.4.0.md

```diff
@@ -0,0 +1,67 @@
+# Release Notes for v0.4.0
+Release Date: 2023-03-03 
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+- Skopio and umoci dependencies are removed with our image-rs component fully integrated
```

Comment:
I'm not sure if we want to call this out in the release notes as we never packaged skopeo and umoci in the CoCo payload, so it was never in a release we produced.

Reply:
ok, it seemed like the dominant thing in the project board. I'm inclined to list it unless your "not sure" goes to "surely not". :-D

Reply:
Typo in skopeo

```suggestion
- Skopeo and umoci dependencies are removed with our image-rs component fully integrated
```

Reply:
It might be better to format the names:
```suggestion
- `skopeo` and `umoci` dependencies are removed with our `image-rs` component fully integrated
```